core/client/tracker/tracker.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tracker verifies consistency proofs
+package tracker
+
+import (
+	"sync"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/proto"
+	"github.com/google/trillian/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	pb "github.com/google/keytransparency/core/api/v1/keytransparency_go_proto"
+	tpb "github.com/google/trillian"
+)
+
+// UpdateTrustedPredicate return a bool indicating whether the local reference
+// for the latest SignedLogRoot should be updated.
+type UpdateTrustedPredicate func(cntRoot, newRoot types.LogRootV1) bool
+
+// LogRootVerifier verifies a Trillian Log Root.
+type LogRootVerifier interface {
+	VerifyRoot(trusted *types.LogRootV1, newRoot *tpb.SignedLogRoot, consistency [][]byte) (*types.LogRootV1, error)
```

Comment:
Please add a comment.
Also, how about renaming `consistency` to `proof`, and clarifying in a comment that it's a consistency proof between the two roots?